### ISSUE TITLE
My Feeds Page

### ISSUE THREAD

**Issue by rudolphpienaar** [FNNDSC/ChRIS_ui#116]

**Comment by rudolphpienaar** [comment]
Switch the link-to-feed from the far right to simply being able to click on the Feed name/icon on the left

**Comment by rudolphpienaar** [comment]
Replace the "view feed details" with different behaviour:

* Feed still running?
* Amount of data/plugins?

**Comment by mairin** [comment]
(Deleted comments transposed into comment #1 for kanban board visibility)

**Comment by Aaishpra** [comment]
Hey, If this is open I would like to work here, as an outreachy internship applicant.

**Comment by PintoGideon** [comment]
Hi @Aaishpra , Firstly, Welcome to the project.

I am currently working on this. A good place to start would be the slack channel with an introduction. There would be an intro to ChRIS talk held early next week to walk you through the project. Post that, we can assign you an open/pending issue. In the meanwhile, feel free to familiarize yourself with the codebase. We use React, Typescript and Patternfly (https://patternfly-react.surge.sh/).

**Comment by Aaishpra** [comment]
Hey thanks for your reply @PintoGideon posted an intro on the matrix,(if that's the correct place to do so)
I have exeperience with using React and typescript but Patternfly is new for me,but seems fun to learn.
If you can provide me a link to slack channel that would be great.

**Comment by Aaishpra** [comment]
Another thing, wouldn't it be better to have a link to chat channels in the readme. It would be better and easier for newcomers to reach out to the community members through these links to the chat channels.
PS: If you think this can be done, I can work on it. 😉

**Comment by mairin** [comment]
> Hey thanks for your reply @PintoGideon posted an intro on the matrix,(if that's the correct place to do so)

Hi, welcome to the ChRIS project @Aaishpra! I do see your intro on the Slack (The Matrix room is linked to the Slack #onboarding room) so it should be the same place.

I am not sure about linking the chat channels in the README because we have so many repos - I wonder if it would make sense to add http://chrisproject.org/join-us to the READMEs of some of the more prominent repos - that is a walkthrough of how to get started, it includes the chat and other stuff too, these might be good repos to do it for:

- https://github.com/FNNDSC/ChRIS_ultron_backEnd
- https://github.com/FNNDSC/ChRIS_ui
- https://github.com/FNNDSC/ChRIS_store_ui
- https://github.com/FNNDSC/ChRIS_store

@Aaishpra ^ that might be a good first start, to update those READMEs with the link if you want to give it a shot

**Comment by Aaishpra** [comment]
@mairin yep, I agree better to add the link you provided on the readme than just chat channel link.
Although I don't see any correct heading to add this link to. Shall I make a new heading something like ` Contact Us` or `Resources` or shall I add it under  current `Notes` heading
